charts/kube-bench-metrics/values.yaml

```diff
@@ -35,28 +35,23 @@ nodeSelector: {}
 
 tolerations: 
       - effect: NoSchedule
-        key: node-role.kubernetes.io/controlplane
-        operator: Equal
-        value: "true"
-      - effect: NoExecute
-        key: node-role.kubernetes.io/etcd
-        operator: Equal
-        value: "true"
+        key: node-role.kubernetes.io/master
+        operator: Exists
 
 affinity: {}
 
-versionOverride: "1.15"
+versionOverride: ""
 
 ## Please use this block to ensure that the /etc/kubernetes path on nodes
 ## are mounted in the daemonSet in readonly mode ##
-rkeEnvironment: true
+rkeEnvironment: false
```

Comment:
Is this used now?

Reply:
Cristian is threatening to bring back rke, or at least that we should support ck8s-apps on top of it

Reply:
Removed the references to rke. Instead added `extraVolumes` / `extraVolumeMounts` as you suggested above.